### ISSUE TITLE
pfm_value_unit not displayed when pfm_range_list is used 

### ISSUE THREAD

**Issue by apizz** [erikberglund/ProfileCreator#111]
While preferences that allow you to enter a numeric value displays a provided `pfm_value_unit` value, it does not appear when there is a dropdown list of values - aka when `pfm_range_list` is used.  I tried adding `pfm_range_list_titles` as well, but the configured `pfm_value_unit` value would still not display.

In the case of this particular preference, because it is slider that snaps to set marked locations along the bar, I wanted to use `pfm_value_unit` to define "Slow" for the low end values and "Fast" for the higher values.

![screen shot 2019-01-02 at 12 22 43 am](https://user-images.githubusercontent.com/11789931/50581126-9e5cbc80-0e24-11e9-9e72-895acce87c04.png)

Example below:
```
                <dict>
			<key>pfm_default</key>
			<integer>11</integer>
			<key>pfm_description</key>
			<string></string>
			<key>pfm_name</key>
			<string>mouseDriverMaxSpeed</string>
			<key>pfm_range_list</key>
			<array>
				<integer>1</integer>
				<integer>2</integer>
				<integer>3</integer>
				<integer>4</integer>
				<integer>6</integer>
				<integer>8</integer>
				<integer>11</integer>
				<integer>19</integer>
				<integer>24</integer>
				<integer>29</integer>
				<integer>32</integer>
			</array>
			<key>pfm_title</key>
			<string>Maximum speed</string>
			<key>pfm_type</key>
			<string>integer</string>
			<key>pfm_value_unit</key>
			<string>Slow - Fast</string>
		</dict>
``` 
![screen shot 2019-01-02 at 12 20 11 am](https://user-images.githubusercontent.com/11789931/50581089-327a5400-0e24-11e9-90c4-18ca5605df16.png)

**Comment by erikberglund** [comment]
Fixed for the next release.

**Comment by apizz** [comment]
Fixed in 0.2.5 release
